okhttp/src/main/java/com/squareup/okhttp/internal/Internal.java

```diff
@@ -70,12 +70,8 @@ public abstract void connectAndSetOwner(OkHttpClient client, Connection connecti
       HttpEngine owner, Request request) throws IOException;
 
   // TODO delete the following when web sockets move into the main package.
-  public abstract Call newCall(OkHttpClient client, Request request);
-  public abstract Response callGetResponse(Call call, boolean forWebSocket) throws IOException;
+  public abstract void callEnqueue(Call call, Callback responseCallback, boolean forWebSocket);
```

Comment:
nit-delux call naming prefix distracted me. no worries as it seems temporary anyway.